packages/kbn-test/src/jest/run.ts

```diff
@@ -73,7 +78,14 @@ export function runJest(configName = 'jest.config.js') {
     process.env.NODE_ENV = 'test';
   }
 
-  run();
+  run().then(() => {
+    // Success means that tests finished, doesn't mean they passed.
+    reportTime(runStartTime, 'total', {
+      success: true,
+      cwd,
+      testFiles,
```

Comment:
There are absolute paths, we should only send the relative path from `REPO_ROOT`